pkg/gits/gitea.go

```diff
@@ -175,17 +175,18 @@ func (p *GiteaProvider) ForkRepository(originalOrg string, name string, destinat
 		if strings.Contains(err.Error(), "try again later") {
 			log.Warnf("Waiting for the fork of %s/%s to appear...\n", owner, name)
 			// lets wait for the fork to occur...
-			start := time.Now()
-			deadline := start.Add(time.Minute)
+			timer := time.NewTimer(time.Minute)
+			defer timer.Stop()
 			for {
-				time.Sleep(5 * time.Second)
-				repo, err = p.Client.GetRepo(owner, name)
-				if repo != nil && err == nil {
-					break
-				}
-				t := time.Now()
-				if t.After(deadline) {
+				select {
```

Comment:
Looking basically ok. Just wondering what your main goal is? Is there a particular issue you want to address or you just want to use more idiomatic Go?

Reply:
Yeah, it just cleanup.
Deprecated ``deadline`` and ``t`` variables, use one variable I think that makes the code cleaner.